src/ray/util/logging.cc

```diff
@@ -114,8 +113,12 @@ void RayLog::StartRayLog(const std::string &app_name, RayLogLevel severity_thres
         app_name_without_path = app_name.substr(pos + 1);
       }
     }
+    google::InitGoogleLogging(app_name_.c_str());
     google::SetLogFilenameExtension(app_name_without_path.c_str());
-    google::SetLogDestination(mapped_severity_threshold, log_dir.c_str());
+    for (enum RayLogLevel level = severity_threshold_; level <= RayLogLevel::FATAL;
+         level = (enum RayLogLevel)((int)level + 1)) {
```

Comment:
1. `enum` is not necessary. 
2. c++-style cast is preferred, say `static_cast`.
3. there could be some problem when you try to case `(int)RayLogLevel::FATAL + 1`. How about:
```
    for (int i = static_cast<int>(severity_threshold_);
         i <= static_cast<int>(RayLogLevel::FATAL); ++i) {
      int level = GetMappedSeverity(static_cast<RayLogLevel>(i));
      google::SetLogDestination(level, dir_ends_with_slash.c_str());
    }
```

Reply:
Okay, thanks!